### PR TITLE
move June 28th to past sessions

### PR DESCRIPTION
**Pull request by bzz**
Archive the prev session.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,11 +23,12 @@ clarifying.
 
 ## Next sessions
 
+- __2019.07.12__ at __4PM CET__ TBD at [src-d/reading-club#68](https://github.com/src-d/reading-club/issues/68)
 
-- __2019.06.28__ at __4PM CET__: [Import2vec  Learning Embeddings for Software Libraries](https://drive.google.com/file/d/1-5VpkX8xo6x6peeccsxD0AgSxx9i2FYv/view?usp=sharing).  We take notes in a [gDrive](https://docs.google.com/document/d/1RHTdZi492x1RfHWYq7ATsyJVzGwfKlOD3RfJquV90Rc/edit?usp=sharing), you are very welcome to ask questions or just give thoughts in it!
+## Past papers
 
+- __2019.06.28__ at __4PM CET__: [Import2vec  Learning Embeddings for Software Libraries](https://drive.google.com/file/d/1-5VpkX8xo6x6peeccsxD0AgSxx9i2FYv/view?usp=sharing).  ([notes](https://docs.google.com/document/d/1RHTdZi492x1RfHWYq7ATsyJVzGwfKlOD3RfJquV90Rc/edit?usp=sharing))
```
Comment (m09) [inline README.md]:
```suggestion
- __2019.06.28__ [Import2vec  Learning Embeddings for Software Libraries](https://drive.google.com/file/d/1-5VpkX8xo6x6peeccsxD0AgSxx9i2FYv/view?usp=sharing). ([notes](https://docs.google.com/document/d/1RHTdZi492x1RfHWYq7ATsyJVzGwfKlOD3RfJquV90Rc/edit?usp=sharing))
```

**Review by m09 — CHANGES_REQUESTED** [review]
Small thing and :+1: Thanks a lot for the admin!

**Comment by bzz** [comment]
@m09 done! Thank you for catching 🚀

**Comment by m09** [comment]
Incredible speed :running_man:

**Review by m09 — APPROVED** [review]

**Pull request merged**